pkg/digger/digger.go

```diff
@@ -141,12 +141,17 @@ func RunCommandsPerProject(commandsPerProject []ProjectCommand, repoOwner string
 		}
 	}
 
+	allAppliesSuccess := true
+
 	for _, success := range appliesPerProject {
 		if !success {
 			allAppliesSuccess = false
 		}
 	}
-	return allAppliesSuccess, nil
+
+	atLeastOneCommandRun := len(commandsPerProject) > 0
+
+	return allAppliesSuccess && atLeastOneCommandRun, nil
```

Comment:
this looks like a hack, you are returning false that the runs were successful which is misleading. Why not do this check explicitly in main.go ? It will be much clearer there

Reply:
not sure what do you mean? There is nothing about successful runs here, only about applies

Reply:
there is nothing saying that all commands are successful? all variables are explicitely called like allAppliesSuccess

Reply:
if I return allAppliesSuccess when there were no projects changes, it's gonna be misleading too

Reply:
I understand the concern tho and will accept a better solution

Reply:
what I can do is to have len(commandsPerProject) > 0 check in main.go and just do an early exit I guess, not just skipping automerge

Reply:
I think checking earlier than the function and exiting with success message is better, "no commands to run, exiting". returning false from function indicates that there were there were some commands and they, at least from my perspective it does not make sense